addons/discuss-button/stick.css

```diff
@@ -0,0 +1,3 @@
+#navigation {

```

Comment:
Could yiu re-add `topnav`?
```suggestion
#navigation, #topnav {

```

Reply:
i didnt even noticed it was removed